README.md

```diff
@@ -1,32 +1,32 @@
-# Ignition tools: Ignition entry point for using all the suite of ignition tools
+# Gazebo tools: Gazebo entry point for using all the suite of Gazebo tools
 
 **Maintainer:** caguero AT openrobotics DOT org
 
-[![GitHub open issues](https://img.shields.io/github/issues-raw/ignitionrobotics/ign-tools.svg)](https://github.com/ignitionrobotics/ign-tools/issues)
-[![GitHub open pull requests](https://img.shields.io/github/issues-pr-raw/ignitionrobotics/ign-tools.svg)](https://github.com/ignitionrobotics/ign-tools/pulls)
+[![GitHub open issues](https://img.shields.io/github/issues-raw/gazebosim/gz-tools.svg)](https://github.com/gazebosim/gz-tools/issues)
+[![GitHub open pull requests](https://img.shields.io/github/issues-pr-raw/gazebosim/gz-tools.svg)](https://github.com/gazebosim/gz-tools/pulls)
 [![Discourse topics](https://img.shields.io/discourse/https/community.gazebosim.org/topics.svg)](https://community.gazebosim.org)
 [![Hex.pm](https://img.shields.io/hexpm/l/plug.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-tools/branch/ign-tools1/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-tools)
-Ubuntu Bionic | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_tools-ci-ign-tools1-bionic-amd64)](https://build.osrfoundation.org/job/ignition_tools-ci-ign-tools1-bionic-amd64)
+Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-tools/branch/ign-tools1/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-tools/branch/ign-tools1)
```

Comment:
We're not uploading test coverage for `gz-tools` at the moment. I think it's mainly because the bulk of the code is Ruby, and we aren't really testing it.

How about we remove the badge for now, and open a ticket to add tests to this library later? Adding tests here should be out of the scope of https://github.com/gazebosim/gz-sim/issues/1575, because there isn't C++ code.

Reply:
Done faa16bf1d42a79028ce5a49ef645212996009b4b & https://github.com/gazebosim/gz-tools/issues/101